hal/src/main/native/athena/Notifier.cpp

```diff
@@ -147,6 +149,11 @@ HAL_NotifierHandle HAL_InitializeNotifier(int32_t* status) {
     std::scoped_lock lock(notifierMutex);
     notifierRunning = true;
     notifierThread = std::thread(notifierThreadMain);
+
+    auto native = notifierThread.native_handle();
+    HAL_SetThreadPriority(&native, notifierThreadRealTime,
```

Comment:
One problem here is the priority status code could get clobbered if tAlarm::create() sets the status. Does tAlarm::create() only set status if there's an error?

Reply:
Generally that's how both our and the NI functions work.  That's why it's important to set status to 0 before the call.  That being said, to be on the safe side you could add a redundant status check.

Reply:
How do we check status codes in the HAL itself?

Reply:
Well, you would just print a message?  You wouldn’t stop the init process of creating the alarm.